<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGA Shape Grammar</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenCGARules"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewShadow"/>
    <addaction name="separator"/>
    <addaction name="actionViewRegularRendering"/>
    <addaction name="actionViewWireframe"/>
    <addaction name="actionViewLineRendering"/>
    <addaction name="actionViewSketchyRendering"/>
    <addaction name="separator"/>
    <addaction name="actionViewRefresh"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionGenerateImages"/>
    <addaction name="actionGenerateBuildingImages"/>
    <addaction name="separator"/>
    <addaction name="actionHoge"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionViewWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionOpenCGARules">
   <property name="text">
    <string>Open CGA Rules</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionViewScopeCoordinateSystem">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scope Coordinate System</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionViewRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionViewShadow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shadow</string>
   </property>
  </action>
  <action name="actionViewLineRendering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Line Rendering</string>
   </property>
  </action>
  <action name="actionGenerateImages">
   <property name="text">
    <string>Generate Images</string>
   </property>
  </action>
  <action name="actionHoge">
   <property name="text">
    <string>Hoge</string>
   </property>
  </action>
  <action name="actionViewRegularRendering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Regular Rendering</string>
   </property>
  </action>
  <action name="actionViewSketchyRendering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sketchy Rendering</string>
   </property>
  </action>
  <action name="actionLSD">
   <property name="text">
    <string>LSD</string>
   </property>
  </action>
  <action name="actionGenerateBuildingImages">
   <property name="text">
    <string>Generate Building Images</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
